archivebox/core/models.py

```diff
@@ -148,3 +151,18 @@ def save_tags(self, tags=[]):
             tags_id.append(Tag.objects.get_or_create(name=tag)[0].id)
         self.tags.clear()
         self.tags.add(*tags_id)
+
+
+class ArchiveResult(models.Model):
+    snapshot = models.ForeignKey(Snapshot, on_delete=models.CASCADE)
+    cmd = models.CharField(max_length=500, default="")
+    pwd = models.CharField(max_length=200, default="")
+    cmd_version = models.CharField(max_length=20, default="")
+    output = models.CharField(max_length=500, default="")
+    start_ts = models.DateTimeField()
+    end_ts = models.DateTimeField()
+    status = models.CharField(max_length=10)
+    extractor = models.CharField(choices=EXTRACTORS, blank=False, max_length=20)
```

Comment:
```suggestion
    extractor = models.CharField(choices=EXTRACTORS, max_length=32)
```
I was taught that there is no point restricting to anything other than a power of 2^n because PostgreSQL, MySQL, and SQLite all bucket columns to the nearest power of two size.

Reply:
Oh really? Didn't know that! Yes, this can be changed.